deepinterpolation/generator_collection.py

```diff
@@ -1109,17 +1111,26 @@ def __data_generation__(self, index_frame):
             # Initialization
             local_path = self.frame_data_location[local_lims]["path"]
 
-            _filenames = ["motion_corrected_video.h5", "concat_31Hz_0.h5"]
-            motion_path = []
-            for _filename in _filenames:
-                _filepath = os.path.join(local_path, "processed", _filename)
-                if os.path.exists(_filepath) and not os.path.islink(
-                    _filepath
-                ):  # Path exists and is not symbolic
-                    motion_path = _filepath
-                    break
-
-            movie_obj = h5py.File(motion_path, "r")
+            motion_path = None
+            if os.path.isfile(local_path):
```

Comment:
@jeromelecoq  This if/else block is confusing. To reduce complexity, can we introduce a breaking change in which the full path is required in the CLI? That way the else block will go away.

Reply:
That will break all of my previous training json files which is annoying. I would vote against because of that as this seems to be marginal technical debt. This PR seems to allow both behavior (pass a full path or a folder).

Reply:
One thing I will say in favor of Adam's position: these names
```
_filenames = ["motion_corrected_video.h5", "concat_31Hz_0.h5"]
```
are no longer the canonical names of the motion corrected movies. They have names like `{ophys_exp_id}_suite2p_motion_corrected_video.h` or somesuch. `concate_31Hz_0.h5` might still exist as a symlink to the other file, but the original logic ignores symlinks, so I don't think that will help you, Jerome.